### DISCUSSION TITLE
Failed to fetch iOS signing files using App Store Connect API

### DISCUSSION THREAD

**Discussion by MattyBoy4444** [codemagic-ci-cd/codemagic-docs#1805] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

My build has been working for a long time.  Now I am getting this error

```
Failed to fetch iOS signing files using App Store Connect API

Executing AppStoreConnect action fetch-signing-files failed unexpectedly. Detailed logs are available at "/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/codemagic-22-06-23.log". To see more details about the error, add `--verbose` command line option.
```

Note: This is different error message than https://github.com/orgs/codemagic-ci-cd/discussions/1801

### Expected results

Make a build

### Actual results

Error making build

### Build id (optional)

6494c86e7f9b8f1e8f0fa33b

**Comment by DorsetDevStudio** [comment]
same here - Apple look to have been updating their UI and auth, my api key was missing but when i clicked 'create' the original code magic one reappeared, will try again tomorrow, hopefully they have fixed it by then.

**Comment by Ashkan-Sarlak** [comment]
I have a similar issue, can't fetch/generate ios certificate or provisioning profile in the "Team Settings".

Fetching "Error":
`Failed to fetch certificates:
Something went wrong`

Generating:
`Something went wrong`

**Comment by isiahmanns** [comment]
I am also getting this. I took a look at the generated log file and in my case, and there is mention of `[SSL: CERTIFICATE_VERIFY_FAILED]`. I generated a new App Store Connect key just for sanity, but that didn't help. Looks like it's an issue with getting a successful connection to Apple.

**Comment by MattyBoy4444** [comment, marked as answer]
It just worked for me.  Apple must have fixed what ever was broke.

**Comment by DorsetDevStudio** [comment]
Working again, problem seems to be solved.
